CHANGELOG.md

```diff
@@ -39,8 +39,29 @@ All notable changes to this project will be documented in this file.
   (<https://github.com/openvinotoolkit/training_extensions/pull/4073>)
 - Upgrade OpenVINO to 2024.5 and NNCF to 2.14.0
   (<https://github.com/openvinotoolkit/training_extensions/pull/4123>)
-- Improve FMetric computation
-  (<https://github.com/openvinotoolkit/training_extensions/pull/4130>)
+
+### Bug fixes
+
+- Fix MaskDINO, MonoDETR recipes
+  (<https://github.com/openvinotoolkit/training_extensions/pull/4080>)
+
+## \[2.3.0\]
```

Comment:
Why do we have two 2.3 releases? Is that a merge artifact?

Reply:
Updated.